src/main/java/com/example/gamelist/GameController.java

```diff
@@ -24,13 +28,23 @@ public  List<Game> gamelist(){
     }
 
 
+
     @PostMapping("/gamelist")
     public ResponseEntity<Response> create(@RequestBody GameForm form){
         URI url = UriComponentsBuilder.fromUriString("http://localhost:8080")
                 .path("/gamelist/id")
                 .build()
                 .toUri();
-        return ResponseEntity.created(url).body(new Response("name successfully created"));
+        return ResponseEntity.created(url).body(new Response("gamelist successfully created"));
+    }
+
+
+
+    @PatchMapping("/gamelist/{id}")
+    public ResponseEntity<UpdateResponse> UpdateGamelist(@PathVariable int id, @RequestBody UpdateRequest updateRequest){
+        gamelistService.updateRequest(id,updateRequest.getName());
+         UpdateResponse updataResponse = new UpdateResponse("Contents have been updated!");
+         return ResponseEntity.ok(updataResponse);
```

Comment:
インデントを揃える意識を持ってください。
```suggestion
         gamelistService.updateRequest(id,updateRequest.getName());
         UpdateResponse updataResponse = new UpdateResponse("Contents have been updated!");
         return ResponseEntity.ok(updataResponse);
```

Reply:
あと , のあとに半角スペースをいれましょう。
```suggestion
         gamelistService.updateRequest(id, updateRequest.getName());
         UpdateResponse updataResponse = new UpdateResponse("Contents have been updated!");
         return ResponseEntity.ok(updataResponse);
```

Reply:
actions on save で自動フォーマットする設定を入れておきさえすればこういった指摘は未然に防げますね。

Reply:
ca7ff6dad314d68be13f1a1031c1f5640945582a

インデントを整え、半角スペースを入れました。
自動フォーマットは入れていたのですが、上手く機能してなかった様です。
改めてチェックを入れて直して行なってみます。
![スクリーンショット 2023-12-26 18 55 29](https://github.com/otyacazyu/gamelist/assets/143002834/1887e5f5-98a2-49cf-872c-7aa221b407e2)

Reply:
OKです。